eq-author/src/App/Comments/index.test.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+
+import { render, queryByTestId } from "tests/utils/rtl";
+
+import CommentsPanel from ".";
+
+/**
+ * Setup module mocks
+ */
+
+const mockUseSubscription = jest.fn();
+const mockUseMutation = jest.fn();
+const mockUseMe = { me: { id: "user-1" } };
+const mockUseQuery = {
+  loading: false,
+  error: false,
+  data: {
+    comments: [
+      {
+        id: "comment-1",
+        user: { id: "user-1", displayName: "Dina" },
+        commentText:
+          "It's all hands on deck, so quit your Myspacing and get on the register. That's an order.",
+        createdTime: "2021-03-30T14:48:00.000Z",
+        editedTime: null,
+        replies: [],
+      },
+      {
+        id: "comment-2",
+        user: { id: "user-1", displayName: "Dina" },
+        commentText:
+          "Has anyone noticed anything out of the ordinary back here? ",
+        createdTime: "2021-03-30T14:49:00.000Z",
+        editedTime: null,
+        replies: [
+          {
+            id: "comment-2-reply-1",
+            user: { id: "user-2", displayName: "Amy" },
+            commentText:
+              "Well, it's Halloween, so everything's kind of out of the ordinary.",
+            createdTime: "2021-03-30T15:01:00.000Z",
+            editedTime: null,
+          },
+        ],
+      },
+    ],
+  },
+};
+
+jest.mock("@apollo/react-hooks", () => ({
+  useMutation: () => [mockUseMutation],
+  useQuery: () => mockUseQuery,
+}));
+
+jest.mock("react-apollo", () => ({
+  useSubscription: () => [mockUseSubscription],
+}));
```

Comment:
```suggestion
jest.mock("@apollo/react-hooks", () => ({
  useMutation: () => [mockUseMutation],
  useQuery: () => mockUseQuery,
    useSubscription: () => [mockUseSubscription],
}));
```
Does this work?

Reply:
I'll give it a try! Watch this space.

Reply:
It does. Madness, nice spot. Cheers dude.